.github/workflows/tests.yml

```diff
@@ -14,29 +14,64 @@ env:
 
 
 jobs:
-  core-and-interface-tests:
+  torch-tests:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        config:
-          - {python-version: 3.8, suite: 'tf'}
-          - {python-version: '3.10', suite: 'tf'}
-          - {python-version: 3.8, suite: 'torch'}
-          - {python-version: '3.11', suite: 'torch'}
-          - {python-version: 3.8, suite: 'jax'}
-          - {python-version: '3.11', suite: 'jax'}
-          - {python-version: 3.8, suite: 'autograd'}
-          - {python-version: 3.9, suite: 'autograd'}
-          - {python-version: '3.10', suite: 'autograd'}
-          - {python-version: '3.11', suite: 'autograd'}
-          - {python-version: 3.8, suite: 'core'}
-          - {python-version: 3.9, suite: 'core'}
-          - {python-version: '3.10', suite: 'core'}
-          - {python-version: '3.11', suite: 'core'}
+        python-version: [3.8, '3.11']

```

Comment:
This is seen throughout the file, but I'm curious as to why there are some tests that only run on limited versions of python. Is it just to prevent too many runners?